### PR TITLE
Test createJob

### LINKED ISSUE #196: Tests for createJob command

### PR DESCRIPTION
**Pull request by Yashk767**
Closes #196

### CODE REVIEW

File: cmd/createJob_test.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+		hash         common.Hash
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    common.Hash
+		wantErr error
+	}{
+		{
+			name: "Test1: When createJob transaction is succesull",
```
Comment (ashish10677) [inline cmd/createJob_test.go:43]: Successful spelling is wrong

File: cmd/createJob.go [hunk 2]
```diff
@@ -31,26 +34,36 @@ Note:
 		power, _ := cmd.Flags().GetInt8("power")
```
Comment (ashish10677) [inline cmd/createJob.go:34]: What happens if no value is provided for power? Add a test case for that as well

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: cmd/createJob.go (hunk 2 above)
Comment (Yashk767) [reply cmd/createJob.go:34]: But we are not checking value of power flag in createJob function, how can I test it .

Comment (ashish10677) [reply cmd/createJob.go:34]: Then you have to handle that in code as well. That is why test case is required.

File: cmd/createJob.go [hunk 3]
```diff
@@ -25,29 +27,48 @@ Note:
...
+		utils.WaitForBlockCompletion(utils.ConnectToClient(config.Provider), txn.String())
 	},
 }
 
-func createJob(txnArgs types.TransactionOptions, power int8, name string, selector string, url string, razorUtils utilsInterface, assetManagerUtils assetManagerInterface, transactionUtils transactionInterface) (common.Hash, error) {
+func createJob(flagSet *pflag.FlagSet, config types.Configurations, razorUtils utilsInterface, assetManagerUtils assetManagerInterface, transactionUtils transactionInterface, flagSetUtils flagSetInterface) (common.Hash, error) {
+	password := razorUtils.AssignPassword(flagSet)
+	address, err := flagSetUtils.GetStringAddress(flagSet)
+	if err != nil {
+		return common.Hash{0x00}, err
```
Comment (ashish10677) [inline cmd/createJob.go:40]: Please use core.NilHash instead of common.Hash{0x00}

**Review by ashish10677 — CHANGES_REQUESTED** [review]

**Comment by ashish10677** [comment]
Can you please report the coverage for this and all subsequent PRs for testing?

**Review by ashish10677 — APPROVED** [review]

**Comment by Yashk767** [comment]
> Can you please report the coverage for this and all subsequent PRs for testing?

With go test -cover ./... , I can only check coverage for respective packages, as we are currently writing tests for cmd so I can report coverage for cmd package with every PR.

**Comment by SkandaBhat** [comment]
Is this up to date with main? We have coveralls on main

**Comment by Yashk767** [comment]
The overall coverage is `5.1%`

Coverage for respective packages : 
```
?       razor   [no test files]
?       razor/accounts  [no test files]
ok      razor/cmd       0.014s  coverage: 21.3% of statements
?       razor/core      [no test files]
?       razor/core/types        [no test files]
?       razor/logger    [no test files]
?       razor/path      [no test files]
?       razor/pkg/bindings      [no test files]
ok      razor/utils     0.144s  coverage: 14.5% of statements


```

**Pull request merged**